### PR TITLE
Add deployment info to gateway_metadata

### PR DESCRIPTION
**Pull request by kurotych**

### CODE REVIEW

File: src/service/mobile_config.proto [hunk 1]
```diff
@@ -21,10 +21,35 @@ import "reward_manifest.proto";
...
+  uint32 azimuth = 3;
+  uint32 mechanical_down_tilt = 4;
+  uint32 electrical_down_tilt = 5;
+}
+
+message cbrs_deployment_info {
+  repeated radio_deployment_info radio_deployment_info = 1;
+}
+
+message radio_deployment_info {
```
Comment (madninja) [inline src/service/mobile_config.proto:38]: Prepend CBRS to this message?

Comment (kurotych) [reply src/service/mobile_config.proto:38]: It makes sense, I will

**Review by jeffgrunewald — APPROVED** [review]

File: src/service/mobile_config.proto [hunk 2]
```diff
@@ -21,10 +21,35 @@ import "reward_manifest.proto";
 // - Keypair fields are binary encoded public keys, Rust encoding example here:
 //    https://github.com/helium/helium-crypto-rs/blob/main/src/public_key.rs#L347-L354
 
+message wifi_deployment_info {
+  // antenna id
+  uint32 antenna = 1;
+  // The height of the hotspot above ground level in whole meters
+  uint32 elevation = 2;
```
Comment (andymck) [inline src/service/mobile_config.proto:28]: maybe add comments for each field here providing some context of each field

**Review by andymck — APPROVED** [review]

File: src/service/mobile_config.proto (hunk 2 above)
Comment (kurotych) [reply src/service/mobile_config.proto:28]: I was thinking about that but nothing comes to mind except obvious: "Antenna azimuth" (etc)

**Pull request merged**